meta-flex-distro/conf/distro/flex-os.conf

```diff
@@ -348,6 +349,11 @@ PUBLIC_REPOS ?= "meta-sokol-flex ${FORKED_REPOS}"
 
 # Define a location for placing external artifacts to be used by the build
 FLEX_EXTERNAL_ARTIFACTS ?= "${TOPDIR}/flex-external-artifacts"
+
+# Add kernel module development capability to SDK
+TOOLCHAIN_HOST_TASK_EXTRA:append = "${@bb.utils.contains('DISTRO_FEATURES', 'flex-debugging', ' nativesdk-flex nativesdk-bison', '', d)}"
+TOOLCHAIN_TARGET_TASK_EXTRA:append = "${@bb.utils.contains('DISTRO_FEATURES', 'flex-debugging', ' flex-kernel-devsrc', '', d)}"
+SDK_POST_INSTALL_COMMAND = "${@bb.utils.contains('DISTRO_FEATURES', 'flex-debugging', '. $target_sdk_dir/environment-setup-${TUNE_PKGARCH}${TARGET_VENDOR}-${TARGET_OS} && cd $target_sdk_dir/sysroots/armv8a-oe-linux/usr/src/kernel && make scripts prepare modules_prepare', '', d)}"
```

Comment:
This is only partially updated. The second command `cd $target_sdk_dir/sysroots/armv8a-oe-linux/usr/src/kernel` should be updated as well.